modules/internal/NetworkAtomics.chpl

```diff
@@ -755,8 +755,8 @@ module NetworkAtomics {
       _v = value;
     }
 
-    proc writeThis(const x) {
```

Comment:
Note that I changed the signature of this `writeThis` to match the others. I think it was just a typo in 41ae1eed844